frameless-runtime/src/utxo.rs

```diff
@@ -0,0 +1,253 @@
+use parity_scale_codec::{Decode, Encode, MaxEncodedLen};
+use scale_info::TypeInfo;
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+use sp_core::{
+	H256,
+	H512,
+	sr25519::{Public, Signature},
+};
+use sp_std::collections::btree_set::BTreeSet;
+use sp_std::prelude::*;
+use sp_runtime::{
+	traits::{BlakeTwo256, Hash},
+	transaction_validity::{TransactionLongevity, ValidTransaction},
+};
+use sp_std::marker::PhantomData;
+
+use log::info;
+
+/// TODO: Clean up this file and organize different parts into different modules for easier reading.
+
+///
+/// TODO: Something similar to construct_runtime! which will setup all the configurations for a UTXO runtime
+/// construct_utxo_runtime!(
+///     MoneyUTXO // Configuration for MoneyUTXO
+///     KittiesUTXO // Configuration for KittiesUTXO
+///     ExistenceUTXO // Configuration for ExistenceUTXO
+/// )
```

Comment:
I definitely agree with this. I think the idea of aggregating pallets/pieces into runtime-level "outer" data types is worth copying from FRAME.